### PR TITLE
Add Alerts Backend Folder To AeroApps

### PR DESCRIPTION
**Pull request by OptimusIacobus**
Add a folder to add functionality in the backend to give an endpoint
for creating alerts using AeroAPI. Also add Docker compose file to run
the alerts backend application.

[JIRA-7000](https://flightaware.atlassian.net/browse/BCK-7000)

### CODE REVIEW

**Review by Dogrock — CHANGES_REQUESTED** [review]

File: alerts_backend/python/app.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    Returns 0 on success, -1 otherwise
+    """
+    try:
+        with engine.begin() as conn:
+            stmt = text("""CREATE TABLE ex_table (fa_alert_id int, ident char(255), origin char(255), destination char(255),
+             aircraft_type char(255), start char(255), end char(255))""")
+            conn.execute(stmt)
+            stmt = text("""INSERT INTO ex_table (fa_alert_id, ident, origin, destination, aircraft_type, start, end)
+             VALUES (:fa_alert_id, :ident, :origin, :destination, :aircraft_type, :start, :end)""")
+            conn.execute(stmt, data_to_insert)
```
[the remark this answers is not in the document]
Comment (OptimusIacobus) [reply alerts_backend/python/app.py:44]: Resolved in #4

File: alerts_backend/python/app.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+def insert_into_db(data_to_insert: dict) -> int:
+    """
+    Insert object into the database based off of the engine
+    Returns 0 on success, -1 otherwise
+    """
+    try:
+        with engine.begin() as conn:
+            stmt = text("""CREATE TABLE ex_table (fa_alert_id int, ident char(255), origin char(255), destination char(255),
+             aircraft_type char(255), start char(255), end char(255))""")
```
[the remark this answers is not in the document]
Comment (OptimusIacobus) [reply alerts_backend/python/app.py:40]: Resolved in #4

[the remark this answers is not in the document]
Comment (OptimusIacobus) [reply alerts_backend/python/app.py:40]: Revoled in #4

**Review by Dogrock — CHANGES_REQUESTED** [review]

File: alerts_backend/python/app.py [hunk 3]
```diff
@@ -69,62 +67,65 @@ def create_alert() -> Response:
...
+            if insert_into_db(data) == -1:
+                r_description = f"Database insertion error, check your database configuration. Alert has still been configured with alert id {r_alert_id}"
+            else:
+                r_success = True
+                r_description = f"Request sent successfully with alert id {r_alert_id}"
+
+    return jsonify({"Alert_id": r_alert_id, "Success": r_success, "Description": r_description})
+
+
+app.run(host="0.0.0.0", port=5001, debug=True)
```
Comment (Dogrock) [inline alerts_backend/python/app.py:131]: I think an accidental port change got through
```suggestion
app.run(host="0.0.0.0", port=5000, debug=True)
```

**Pull request merged**